generator/index.js

```diff
@@ -4,15 +4,14 @@ const rimraf = require('rimraf');
 module.exports = (api, options, rootOptions) => {
 
   const pathsToRemove = [
-    'src/router.js',
-    'src/store.js',
```

Comment:
Are these files no longer generated by Vue CLI?

Reply:
Nope, at least with default selection.
This is what it generates:
![image](https://user-images.githubusercontent.com/2633898/100094323-20ab5b00-2e59-11eb-97bc-d3d0d813fba7.png)